workspaces/diff-engine/tests/fixtures/shape-diff-use-cases/deeply nested fields inside of arrays.json

```diff
@@ -1,2637 +1,2571 @@
 {
```

Comment:
Ok so during my merge commit, prettier picked this up since it's considered a staged commit - this should be identical just format changes